lucene/core/src/java/org/apache/lucene/search/SearcherLifetimeManager.java

```diff
@@ -238,7 +238,7 @@ public boolean doPrune(double ageSec, IndexSearcher searcher) {
    *  entries are passed to the Pruner in sorted (newest to
    *  oldest IndexSearcher) order.
    * 
-   *  <p><b>NOTE</b>: you must peridiocally call this, ideally
+   *  <p><b>NOTE</b>: you must periodiocally call this, ideally
```

Comment:
This should be `periodically`, right?

Reply:
+1 will try rebasing it later today/this week